### PR TITLE
Fix typo in bit-vector-copy docs

### PR DESCRIPTION
**Pull request by williewillus**
Should be bit-vector-length not vector-length

### CODE REVIEW

File: data-doc/data/scribblings/bit-vector.scrbl [hunk 1]
```diff
@@ -101,7 +101,7 @@ Returns the number of set bits in the bit-vector @racket[bv].
 
 @defproc[(bit-vector-copy [bv bit-vector?]
                           [start exact-nonnegative-integer? 0]
-                          [end exact-nonnegative-integer? (vector-length v)])
+                          [end exact-nonnegative-integer? (bit-vector-length v)])
```
Comment (sorawee) [inline data-doc/data/scribblings/bit-vector.scrbl:104]: Perhaps `v` should also be changed to `bv`?

**Comment by sorawee** [comment]
You might also want to fix `bit-vector-set!`'s `vv` to `bv`.

**Comment by williewillus** [comment]
Done, thanks!

**Comment by sorawee** [comment]
This looks good to me.

**Pull request merged**